components/Form/MultiStepForm.js

```diff
@@ -112,7 +112,7 @@ export class MultiStepForm extends React.Component {
         onSubmit={this.handleSubmit}
         render={formikBag => (
           <Form className={styles.MultiStepForm} onSubmit={formikBag.handleSubmit}>
-            <CurrentStep {...formikBag} />
+            <CurrentStep {...formikBag} stepNumber={stepNumber} totalSteps={steps.length} />
```

Comment:
This a good solution!

Now that you've done this, I think I'd like for each step to care less about the progress indicator, and have the progress bar itself get rendered in this (MultiStepForm) component. To do that, we'd need to make the form title into a static property for each step - that would actually reduce code a bit! Then we can do:

```jsx
<h3>{CurrentStep.title}</h3>

<ProgressIndicator stepNumber={stepNumber} totalSteps={steps.length} />

<CurrentStep {...formikBag} />
```